proposed/http-middleware/middleware-meta.md

```diff
@@ -123,7 +120,7 @@ Middleware taking this approach generally has the following commonalities:
   the request for processing.
 * The middleware is passed 2 arguments during invocation:
   1. An object that represents an HTTP request.
-  2. A delegate that receives the request to dispatch next middleware in the pipeline.
+  2. A delegate that receives the request to dispatch next middleware in the stack.
```

Comment:
Yep, I read the post.
While I agree that the reference to the "pipeline processing pattern" wikipedia page may result confusing, I think using `stack` instead of `pipeline` is plain wrong, because there may be no stack involved at all: the data structure used to handle a group of middlewares is an implementation detail.
Now the generic `pipeline` word doesn't suggest any type of specific data structure in this regard, while `stack` does.